.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        REACT_DIST: [16, 17]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

The current active LTS version of Node is 16, so v16 seems to be better.

https://nodejs.org/en/

```suggestion
      - name: Use Node.js 16
        uses: actions/setup-node@v2
        with:
          node-version: 16
```

---

The goal of this PR was to move from Travis to GitHub actions. Travis was running on node 14. I think we should do this in a follow-up.